test/unit/search/fm_index_cursor/fm_index_cursor_collection_test_template.hpp

```diff
@@ -156,10 +185,20 @@ TYPED_TEST_P(fm_index_cursor_collection_test, extend_right_char)
     EXPECT_TRUE(it.extend_right(seqan3::views::slice(this->text1, 0, 1)));   // "A"
     EXPECT_EQ(seqan3::uniquify(it.locate()), (std::vector<std::pair<uint64_t, uint64_t>>{{0,0}, {0,3}, {1,4}, {1,8}}));
     EXPECT_EQ(it.query_length(), 1u);
+    if constexpr (!seqan3::bi_fm_index_specialisation<typename TypeParam::index_type>)
+    {
+        EXPECT_EQ(it.left_bound(), 1u);
+        EXPECT_EQ(it.right_bound(), 4u);
+    }
 
     EXPECT_TRUE(it.extend_right(seqan3::views::slice(this->text1, 1, 2)));   // "C"
     EXPECT_EQ(seqan3::uniquify(it.locate()), (std::vector<std::pair<uint64_t, uint64_t>>{{0,0}, {0,3}}));
     EXPECT_EQ(it.query_length(), 2u);
+    if constexpr (!seqan3::bi_fm_index_specialisation<typename TypeParam::index_type>)
+    {
+        EXPECT_EQ(it.left_bound(), 5u);
+        EXPECT_EQ(it.right_bound(), 6u);
```

Comment:
If bidirectional would implement this, would it return the same values?

Reply:
I would delegate the `left_bound` on a `bi_fm_index` to the `left_bound` of the forward `fm_index`, so it should return the same values.